### ISSUE TITLE
Question: how to create an alias for a Rust dependency?

### ISSUE THREAD

**Issue by vanc** [mesonbuild/meson#11776]
I'm following the example in [this repository](https://gitlab.freedesktop.org/slomo/gst-plugin-meson-test) to integrate external crates into my project. However, when porting the [OpenSSL crate](https://crates.io/crates/openssl), I got an issue. The openssl crate uses aliasing for the `openssl-sys` crate as `ffi`. It does that in its Cargo.toml:

```
[dependencies]
...
ffi = { package = "openssl-sys", version = "0.9.87", path = "../openssl-sys" }
```

With Cargo, it basically uses the same '--extern', but replaces 'openssl_sys' with 'ffi'. I don't see a way to do that in Meson.

Any suggestions?

**Comment by eli-schwartz** [comment]
That repository includes some features which the author had to submit PRs to meson to implement support for. This one specifically was implemented in #11681 and documented in the release notes at https://mesonbuild.com/Release-notes-for-1-2-0.html#support-for-defining-crate-names-of-rust-dependencies-in-rust-targets

You'll need to run meson from git in order to do this, for now.

**Comment by vanc** [comment]
Thanks for the pointer.
